contracts/libraries/TransferHelper.sol

```diff
@@ -3,6 +3,10 @@ pragma solidity ^0.8.15;
 
 import {IERC20Minimal} from "@uniswap/v4-core/contracts/interfaces/external/IERC20Minimal.sol";
 
+error TransferFailed();
+error STF();
+error STE();
```

Comment:
now that we use custom errors we can make these more descriptive (they were acronyms before to save on bytes!)

maybe `error SafeTransferFailed` and `error SafeTransferEthFailed`

Reply:
oo leaving a TODO here to remove `TransferHelper` to use solmate ver after we migrate twamm to it too